### PR TITLE
Corrected zotero icon aliase and added class for accessibility check

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
+ Correct the zotero iconURL aliases

+ Added unoAccessibilityCheck class
  - added class for Accessibility option inside review tab
  - needed for integration with different theme.

### CODE REVIEW

File: browser/src/control/Control.JSDialogBuilder.js [hunk 1]
```diff
@@ -2176,7 +2176,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			'zoteroaddnote': 'addcitationnote',
 			'zoterorefresh': 'updateall',
 			'zoterounlink': 'unlinkcitation',
-			'zoteroaddeditbibliography': 'addeditbibliography',
```
Comment (eszkadev) [inline browser/src/control/Control.JSDialogBuilder.js:2179]: I see that `zoteroaddeditbibliography` is also used somewhere, are you sure we can remove it? shouldn't we keep 2 of them?

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.JSDialogBuilder.js:2179]: @eszkadev i have not seen zoteroaddeditbibliography at any place i did test with NC intergration

Comment (eszkadev) [reply browser/src/control/Control.JSDialogBuilder.js:2179]: `git grep zoteroaddeditbibliography` shows usage, maybe it is only shown under some conditions
do you have zotero configured? (API token)

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.JSDialogBuilder.js:2179]: No i do not have zotero API token. let me try with NC integration with local branding

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.JSDialogBuilder.js:2179]: @eszkadev just tested with local theme with Zotero it is working okay. We are good to replace this keyword

Comment (eszkadev) [reply browser/src/control/Control.JSDialogBuilder.js:2179]: no, please leave it. If it is here and somewhere is used in the code - let's trust we need it.

grep:
```
browser/src/control/Control.JSDialogBuilder.js:                 'zoteroaddeditbibliography': 'addeditbibliography',
browser/src/control/Control.Menubar.js:                         {name: _('Add Bibliography'), id: 'zoteroaddeditbibliography', type: 'action', hidden: !window.zoteroEnabled},
browser/src/control/Control.NotebookbarWriter.js:                                       'id': 'zoteroaddeditbibliography',
browser/src/control/Control.NotebookbarWriter.js:                                       'class': 'unozoteroaddeditbibliography',
browser/src/docdispatcher.ts:           this.actionsMap['zoteroaddeditbibliography'] = function () {
```

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.JSDialogBuilder.js:2179]: Sure i have added the original value and also the corrected one.

**Review by eszkadev — APPROVED** [review]

**Pull request merged**